homeassistant/components/renault/renault_coordinator.py

```diff
@@ -17,9 +23,23 @@
 class RenaultDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
Maybe it should be this?
```python
T = TypeVar("T")
class RenaultDataUpdateCoordinator(Generic[T], DataUpdateCoordinator[T]):
```

Reply:
Or just this. Then you don't need to make it generic and use `T`.
```python
class RenaultDataUpdateCoordinator(DataUpdateCoordinator[CoordinatorReturnType]):
```

Reply:
Should be fixed in c6e47480e432ad875000f9b405812bfaddbb37e1